### ISSUE TITLE
/data/config/db path isn't created by minikube setup 

### ISSUE THREAD

**Issue by psionikangel** [GluuFederation/gluu-docker#89]
Hello,

I am trying to follow the setup instructions for a minikube setup on my machine and I can't complete all the steps.

**Environment**
minikube version : 1.4.0 (A fresh install done this week)

**Description of the issue:**

The path `/data/config/db` is not found

**Steps to reproduce**

1. I start with a brand new k8s v 1.16.0 cluster with `minikube start`

2. Follow the steps exactly as shown in the minikube readme

**Expected behavior**
I would expect following the steps as-is to produce a clean and working setup

**Actual behavior**
The following step is problematic `kubectl apply -f generate-config.yaml`. It seems the persistent volume claim doesn't create the /data/config/db path on the host

**Additional information**
This is the events section of the output of `kubectl describe pods` after running the generate config command

```
Events:
  Type     Reason     Age                   From               Message
  ----     ------     ----                  ----               -------
  Normal   Scheduled  <unknown>             default-scheduler  Successfully assigned default/config-init-7cghw to minikube
  Normal   Pulling    2m53s                 kubelet, minikube  Pulling image "gluufederation/config-init:3.1.6_03"
  Normal   Pulled     2m26s                 kubelet, minikube  Successfully pulled image "gluufederation/config-init:3.1.6_03"
  Warning  Failed     15s (x12 over 2m26s)  kubelet, minikube  Error: stat /data/config/db: no such file or directory
  Normal   Pulled     15s (x11 over 2m26s)  kubelet, minikube  Container image "gluufederation/config-init:3.1.6_03" already present on machine
```

**Comment by iromli** [comment]
Hi @psionikangel,

Try adding the following lines in `config-volumes.yaml`:

```
hostPath:
  path: "/data/config/db"
  type: DirectoryOrCreate
```

Let me know the result.

**Comment by psionikangel** [comment]
Hi @iromli,

Adding the `DirectoryOrCreate` type did fix the issue and I was able to continue with the setup in minikube. Thank you for that.

After following along the readme I was able to get a gluu server up and running in minikube but there was a few minor modifications I had to do to make it happen. Would you guys be open to a pull request that adds a bit of information to the readme as well as the small modifications I made to the `.yaml` files?

**Comment by mogluu** [comment]
Hey @psionikangel,

We have all that fixed in our new 4.0.0 version. You may submit a pull request for review.

Thanks,

**Comment by iromli** [comment]
Closing this issue ... feel free to submit PR
